package/autoyast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Sep 30 10:21:45 CEST 2016 - schubi@suse.de
+
+- Adding missed desktop file for "clone_system" in order to show
+  it in the control center and command line calls.
+  (bnc#985621)
+- 3.1.152
```

Comment:
BTW this change is already in master as 3.1.152, but master has since progressed to 3.2.0, so there is no danger of releasing same-versioned packages with different changes.
(CASP is using 3.1.151.1)